qa/setup_packages.py

```diff
@@ -411,14 +458,16 @@ def cal_num_of_configs(packages, cuda_version):
             ret *= pckg.get_num_of_version(cuda_version)
     return ret
 
-def for_all_pckg(packages, fun):
-    """Iterates over all packages, executes a fun returns all fun results as a list"""
+def for_all_pckg(packages, fun, add_additional_packages=True):
+    """Iterates over all packages, executes a fun. Returns all fun results as a list"""
```

Comment:
```suggestion
    """Iterates over all packages, executes a function. Returns all function results as a list"""
```
"All fun results" read weird :)

Reply:
Done